### DISCUSSION TITLE
Java Client SDK

### DISCUSSION THREAD

**Discussion by ItsFunny** [cosmos/community#94] [Clients (JS)]
### Summary

any repo for java client develper?

**Comment by facundomedica** [comment]
Hey @ItsFunny you have the proto files that you can use to generate the client. For queries it should be fairly simple, for sending transactions you'll need to have a way of signing. We don't provide out of the box support for Java, but you should be able to replicate the signing process.

I'll leave this open for now in case someone has a Java client they want to suggest

**Comment by julienrbrt** [comment]
Converting this to a discussion
